packages/backend/src/modules/dashboard/dashboard.repository.ts

```diff
@@ -1,15 +1,49 @@
-import { CreateTaskCommandDTO } from 'shared'
-import DashboardModel from './Dashboard'
+import { CreateTaskCommandDTO, CreateDashboardCommand } from 'shared'
+import mongoose from 'mongoose'
+import { Dashboard as DashboardModel } from './dashboard.model'
 import { Task } from '../task/task'
+import { Dashboard } from './Dashboard'
 
 export class DashboardRepository {
-  private readonly _dashboardModel = DashboardModel
+  private readonly _dashboardModel
 
-  constructor(boardModel: any) {
-    this._dashboardModel = boardModel
+  constructor(dashboardModel: any) {
```

Comment:
You can make it, same as [here ](https://github.com/kacper-cyra/camp-trello-clone/pull/113/files#diff-44f96226e0025aa1eb9c81007908cc30c0e8c76ea08c893de43963b5e566f9a7)to avoid any

Reply:
The link leads to a file in which this piece of code looks the same